distribution/changelog.txt

```diff
@@ -33,6 +33,7 @@
 - Fix: [#5585] Inconsistent zooming with mouse wheel.
 - Fix: Infinite loop when removing scenery elements with >127 base height.
 - Fix: Ghosting of transparent map elements when the viewport is moved in OpenGL mode.
+- Fix: Save prompt sometimes proposes wrong save name.
```

Comment:
This is a change that was introduced in the same version, we don't put those in changelog

Reply:
Sorry, I didn't check for that.